### ISSUE TITLE
Scenario Page: Login and Registration

### ISSUE THREAD

**Issue by davidortinau** [davidortinau/Xappy#10]
# Description

- User can enter a username and password (obscured)
- Email should be validated
- Password should be validated (length)
- User can tap Login and see some transition/response
- User can go to registration state
- User can enter username, password, and password confirmation with rule validation, error states, and password strength indicator

![SignUp](https://user-images.githubusercontent.com/41873/56399648-076e1180-6215-11e9-9647-22324de73242.png)
![Login](https://user-images.githubusercontent.com/41873/56399649-076e1180-6215-11e9-9109-73033bca02dc.png)

## Sub Tasks

- [ ] Do This
- [ ] Do That

**Comment by HobDev** [comment]
@davidortinau  I want to work on this. Please guide me in the right direction. 
Thank you

**Comment by robintschroeder** [comment]
@HobDev Are you actively working on this?

**Comment by HobDev** [comment]
No, as the work is not assigned and I am not sure that anyone else is not working on this.

**Comment by robintschroeder** [comment]
Hi @davidortinau - I can work on these two pages.

**Comment by robintschroeder** [comment]
Looks like Cyril Cathala has completed this issue? https://www.youtube.com/watch?v=g07N-E05AYA
